### ISSUE TITLE
Geometric Mean Metric Learning

### ISSUE THREAD

**Issue by sgnoya** [ComputerVisionLaboratory/survey#5]
## 1. 概要（基本アイデア）
- マハラノビス距離に基づく計量学習法，Geometric Mean Metric Learning; GMML, を提案．
- 定義した損失関数の強凸性とstrictly geodesically convex（強測地凸？）性を示すことで，最適解が損失の勾配=0となる点であることを示した
- 更に，損失関数の勾配=0と置くと Riccati 方程式が得られるため，簡単に最適解が求められることを示した

## 2. 新規性
計量学習の定式化としては一般的なものに見える．
最適解の一意性と，最適解が微分方程式で簡単に得られることを示したことがスゴイ

上記を示すため，非類似(異クラス)間の距離を計量行列Ａの逆行列で計算されるinterpoint distanceで定義しなおしたところが大事そう

## 3. 手法詳細

損失関数は，類似ペアセット(SP)と非類似ペアセット(DP)の距離/inter point distanceの和：
\sum_{ij\in SP} (x_i-x_j)^T A (x_i-x_j) + \sum_{ij\in DP} (x_i-x_j)^T A^{-1} (x_i-x_j)

最適解は，クラス間分散行列Dとクラス内分散行列Sの逆行列(S^{-1})間のSPD多様体上での測地線の中点！

また，DとSをtでバランスした場合(loss = t D + (1- t) S),  最適解はDからS^{-1}への測地線上でtだけ動いた点！

## 4. 結果
識別結果自体は従来法とそれほど変わらないが，計算時間がものすごく短くなっている．

## 5. 論文，コード等へのリンク
[ICML site](http://proceedings.mlr.press/v48/zadeh16.html)
[Matlab Code](https://github.com/PouriaZ/GMML)

## 6. 感想，コメント
解がきれいすぎて感動した

## 7. bibtex
@inproceedings{zadeh16geometric,
author = {Zadeh, Pourya Habib and Hosseini, Reshad and Sra, Suvrit},
title = {Geometric mean metric learning},
booktitle={Proceedings of the 33rd international conference on Machine learning},
year = {2016}
}
